### DISCUSSION TITLE
Disable Hit Counter on Relax plays.

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#21158] [Ideas] [answered]
Look at this profile, please: https://osu.ppy.sh/users/8118575
This player playing maps with relax on 20-40% accuracy (11 hours ago games). It looks like stats grinding - 5 000 000 hits with only 14 000 playcount ( I have 10M with 43k playcount, for example ). It would be better to disable hit counter to prevent cases like this.

**Comment by peppy** [comment, marked as answer]
Please report the user via their profile. We will deal with this as a foul play case.

**Reply by okolenmi** [reply]
I'm talking about Relax mode. This user just for example. I don't care about him. But the fact that relax plays affect Hit Count is strange.
